documentation/leo/03_language.md

```diff
@@ -281,6 +287,24 @@ function foo(
 }
 ```
 
+### Inline Function
+
+An inline function is declared as `inline {name}() {}`.
+Inline functions contain expressions and statements that can compute values.
+Inline functions that cannot be executed directly, 
+instead the Leo compiler “inlines” the body of the function at each call site.
```

Comment:
You could omit the double quotes around "inlines": it's a proper technical term.